bootstrap/bootstrap.sh

```diff
@@ -0,0 +1,268 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# See LICENSE file in the project root for full license information.
+
+# Note: This script should be compatible with the dash shell used in Ubuntu. So avoid bashisms! See https://wiki.ubuntu.com/DashAsBinSh for more info
```

Comment:
Actually, could you clarify this comment/note? Most of our other scripts are written for bash, and I think most folks use bash as their shell, too. Unless I'm misunderstanding how shebangs work (definitely possible 😄 ).